Article reviewed

biaguerreiro opened the pull request:
"Por que a autenticação utilizando appKey e appToken não funciona?" reviewed

biaguerreiro committed:
Article reviewed

"Por que a autenticação utilizando appKey e appToken não funciona?" reviewed

phdro88 commented on articles/pt/por-que-a-autenticacao-utilizando-appkey-e-apptoken-nao-funciona.md:
```diff
@@ -0,0 +1,34 @@
+Na VTEX, a autenticação das [APIs](/tutorial/introducao-as-apis-vtex--3SjAqQ0BeUqu2ge8AiIkmW) para integrações com a sua loja é realizada por meio de [appKey e appToken](/tutorial/criar-appkey-e-apptoken-para-autenticar-integracoes--43tQeyQJgAKGEuCqQKAOI2). 
```
Eu trocaria aqui a ordem dos termos:

"Na VTEX, a autenticação de integrações via [APIs](/tutorial/introducao-as-apis-vtex--3SjAqQ0BeUqu2ge8AiIkmW) é realizada por meio de [appKey e appToken](/tutorial/criar-appkey-e-apptoken-para-autenticar-integracoes--43tQeyQJgAKGEuCqQKAOI2)."

phdro88 commented on articles/pt/por-que-a-autenticacao-utilizando-appkey-e-apptoken-nao-funciona.md:
```diff
@@ -0,0 +1,34 @@
+Na VTEX, a autenticação das [APIs](/tutorial/introducao-as-apis-vtex--3SjAqQ0BeUqu2ge8AiIkmW) para integrações com a sua loja é realizada por meio de [appKey e appToken](/tutorial/criar-appkey-e-apptoken-para-autenticar-integracoes--43tQeyQJgAKGEuCqQKAOI2). 
+
+ Existem alguns casos em que, por conta de erros relacionados a essas chaves de autenticação, a integração falha.  
```
Sempre que possível, prefira a ordem direta nas frases:

"Existem alguns casos em que uma integração pode falhar por conta de erros relacionados a essas chaves de autenticação."

Eu também colocaria esta frase no parágrafo anterior. Da maneira que está, temos dois parágrafos seguidos com uma linha apenas.

phdro88 commented on articles/pt/por-que-a-autenticacao-utilizando-appkey-e-apptoken-nao-funciona.md:
```diff
@@ -0,0 +1,34 @@
+
+ ## Associação de appKey e appToken
+
+ Ao acessar __Perfis de acesso__, dentro do módulo de __[Gerenciamento da Conta](https://help.vtex.com/tutorial/visao-geral-do-modulo-license-manager?locale=pt)__, é possível [configurar perfis](https://help.vtex.com/tutorial/visao-geral-do-modulo-license-manager?locale=pt) com níveis de acesso diferentes à plataforma.
```
Trocar o link de "configurar perfis" para /tutorial/como-criar-perfil-de-acesso--tutorials_526

phdro88 commented on articles/pt/por-que-a-autenticacao-utilizando-appkey-e-apptoken-nao-funciona.md:
```diff
@@ -0,0 +1,34 @@
+
+ Existem alguns casos em que, por conta de erros relacionados a essas chaves de autenticação, a integração falha.  
+
+ Verifique a seguir três cenários para garantir uma autenticação bem-sucedida:
```
Achei que a linguagem ficou meio formal aqui. Escreveria assim:

"Veja abaixo como solucionar alguns problemas comuns relacionados à autenticação por appKey e appToken"

Eu também colocaria esse frase junta com as duas anteriores, criando um primeiro parágrafo de três frases. As três frases estão dentro do mesmo contexto, não é necessário separá-las em parágrafos diferentes.

biaguerreiro committed:
article updated. 

por-que-a-autenticacao-utilizando-appkey-e-apptoken-nao-funciona reviewed and approved.
